client/lib/react-helpers/index.js

```diff
@@ -35,7 +36,11 @@ export function renderWithReduxStore( reactElement, domContainer, reduxStore ) {
 			: domContainer;
 
 	return ReactDom.render(
-		React.createElement( ReduxProvider, { store: reduxStore }, reactElement ),
+		<ReduxProvider store={ reduxStore }>
+			<Stylizer onInsertCss={ insertCss }>
```

Comment:
Do we need it for every page or can we put it in the top component like `<Layoyt />`?